libs/extensions/src/NanoEvent.cpp

```diff
@@ -68,11 +68,11 @@ shared_ptr<PhysicsObjects> NanoEvent::GetSegmentMatchedMuons(float minMatchRatio
     float nHits = float(dsaMuon->Get("nSegmentHits"));
     
     bool matchFound = false;
-    float ratio_tmp = float(dsaMuon->Get("muonMatch1")) / nHits;
     for(int i=1; i<=5; i++) {
       float ratio_tmp = asNanoMuon(dsaMuon)->GetMatchesForNthBestMatch(i) / nHits;
       if(!matchFound && ratio_tmp >= minMatchRatio) {
-        if(MuonIndexExist(looseMuons, asNanoMuon(dsaMuon)->GetMatchIdxForNthBestMatch(i)),true) matchFound = true;
+        if(MuonIndexExist(looseMuons, asNanoMuon(dsaMuon)->GetMatchIdxForNthBestMatch(i),false)) matchFound = true;
```

Comment:
I'm still amazed C++ allows what was here before... nice catch! :)

Reply:
Actually, wouldn't it be cleaner to simply do:

```C++
matchFound = PATMuonIndexExist(looseMuons, asNanoMuon(dsaMuon)->GetMatchIdxForNthBestMatch(i));
```

Reply:
Yep I already changed this :)

Reply:
... now ;)